doc/user_manual/existing_interfaces.tex

```diff
@@ -2047,3 +2044,327 @@ \subsubsection{Steps}
+    <Sampler class='Samplers'  type='Grid'>Grid_Sampler</Sampler>
+    <Output  class='Databases' type='HDF5'>Grid_out</Output>
+    <Output  class='DataObjects' type='PointSet'   >GridMelcorPointSet</Output>
+    <Output  class='DataObjects' type='HistorySet'>GridMelcorHistorySet</Output>
+  </MultiRun>
+  <MultiRun name='MC_Sampler' verbosity='debug' re-seeding='210491'>
+    <Input   class='Files' type=''>inputFileMelcor.i</Input>
+    <Input   class='Files' type=''>restartFile</Input>
+    <Model   class='Models'    type='Code'>MyMELCOR</Model>
+    <Sampler class='Samplers'  type='MonteCarlo'>MC_Sampler</Sampler>
+    <Output  class='Databases' type='HDF5'      >MC_out</Output>
+    <Output  class='DataObjects' type='PointSet'   >MonteCarloMelcorPointSet</Output>
+    <Output  class='DataObjects' type='HistorySet'>MonteCarloMelcorHistorySet</Output>
+  </MultiRun>
+</Steps>
+\end{lstlisting}
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsubsection{Databases}
+As shown in the \xmlNode{Steps} block, the code is creating two database objects
+called \texttt{Grid\_out} and \texttt{MC\_out}.
+%
+So the user needs to input the following:
+\begin{lstlisting}[style=XML]
+<Databases>
+  <HDF5 name="Grid_out" readMode="overwrite"/>
+  <HDF5 name="MC_out" readMode="overwrite"/>
+</Databases>
+\end{lstlisting}
+As listed before, this will create two databases.
+%
+The files will have names corresponding to their \xmlAttr{name} appended with
+the .h5 extension (i.e. \texttt{Grid\_out.h5} and \texttt{MC\_out.h5}).
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsubsection{DataObjects}
+As shown in the \xmlNode{Steps} block, the code is creating $4$ data objects ($2$ HistorySet and $2$ PointSet)
+called \texttt{GridMelcorPointSet} \texttt{GridMelcorHistorySet} \texttt{MonteCarloMelcorPointSet} and
+ \texttt{MonteCarloMelcorHistorySet}.
+%
+So the user needs to input the following block as well, where the Input and Output variables are listed:
+\begin{lstlisting}[style=XML]
+  <DataObjects>
+    <PointSet name="GridMelcorPointSet">
+      <Input>PRE</Input>
+      <Output>
+        time,volume_1_PRESSURE,volume_1_TLIQ,
+        volume_1_TVAP,volume_1_MASS
+      </Output>
+    </PointSet>
+    <HistorySet name="GridMelcorHistorySet">
+      <Input>PRE</Input>
+      <Output>
+        time,volume_1_PRESSURE,volume_1_TLIQ,
+        volume_1_TVAP,volume_1_MASS
+      </Output>
+    </HistorySet>
+    <PointSet name="MonteCarloMelcorPointSet">
+      <Input>PRE</Input>
+      <Output>
+        time,volume_1_PRESSURE,volume_1_TLIQ,
+        volume_1_TVAP,volume_1_MASS
+      </Output>
+    </PointSet>
+    <HistorySet name="MonteCarloMelcorHistorySet">
+      <Input>PRE</Input>
+      <Output>
+        time,volume_1_PRESSURE,volume_1_TLIQ,
+        volume_1_TVAP,volume_1_MASS
+      </Output>
+    </HistorySet>
+  </DataObjects>
+\end{lstlisting}
+As mentioned before, this will create $4$ DataObjects.
+%
```

Comment:
General comment, this is useful documentation, but it seems like it is duplicating information elsewhere in the user manual.  Possibly one solution would move most of this to the user guide or somewhere else.  Fixing this is just a suggestion.